_opensearch/query-dsl/text-analyzers.md

```diff
@@ -0,0 +1,104 @@
+---
+layout: default
+title: Text analyzers
+parent: Query DSL
+nav_order: 41
+---
+
+
+# Optimizing text for searches with text analyzers
+
+OpenSearch applies text analysis during indexing or searching for `text` fields. There is a standard  analyzer that OpenSearch uses by default for text analysis. To optimize unstructured text for search, you can convert it into structured text with our text analyzers.
+
+## Text analyzers
+
+OpenSearch provides several text analyzers to convert your structured text into the format that works best for your searches.
+
+OpenSearch supports the following text analyzers:
+
+1. **Standard analyzer** – parses strings into terms at word boundaries per the Unicode text segmentation algorithm. It removes most, but not all punctuation. It converts strings to lowercase. You can remove stop words if you turn on that option, but it does not remove stop words by default.
```

Comment:
All list items: Capitalize the first word following the en dash. Insert a comma after "all".